kubernetes/gke-base/docs/index.rst

```diff
@@ -0,0 +1,430 @@
+	statefulset.apps/schemaregistry   1         1         9m59s
+	statefulset.apps/zookeeper        1         1         11m
+
+	NAME                                        COMPLETIONS   DURATION   AGE
+	job.batch/clicks-datagen-connector-deploy   1/1           4s         8m6s
+
+	NAME                                               AGE
+	zookeepercluster.cluster.confluent.com/zookeeper   11m
+
+	NAME                                       AGE
+	kafkacluster.cluster.confluent.com/kafka   10m
+
+Verify Confluent Platform on the CLI 
+````````````````````````````````````
+
+By default the demo is deployed without any `Kubernetes Ingress <https://kubernetes.io/docs/concepts/services-networking/ingress/>`__, which means the |cp| resources inside the Kubernetes cluster cannot be reached from external clients.  If you used a pre-existing cluster with Ingress enabled, the following validation instructions may not be applicable to your setup.
+
+The demo deploys a ``client-console`` pod that can be used to open a terminal inside the cluster with network connectivity to the |cp| services.  For example::
+
+	kubectl -n operator exec -it client-console bash
+
+From here you can execute standard |ak| commands to validate the cluster.  You need to provide the commands with the required connectivity and security configurations, which are provided in mapped files on the ``client-console`` pod.  See the :ref:`examples-operator-gke-base-client-configurations` Highlight for more information.
+
+.. sourcecode:: bash
+
+		kafka-topics --bootstrap-server kafka:9071 --command-config /etc/kafka-client-properties/kafka-client.properties --list
+
+You could view the output of the random click data generator with the console consumer::
+
+	kafka-console-consumer --bootstrap-server kafka:9071 --consumer.config /etc/kafka-client-properties/kafka-client.properties --topic clicks
+
+Example output might look like::
+
+	222.152.45.45F-
+	16141<GET /images/track.png HTTP/1.1204006-Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36
+	122.173.165.203L-
+	16151FGET /site/user_status.html HTTP/1.1401289-Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
+	...
+
+Verify Confluent Platform Control Center
+````````````````````````````````````````
+
+In order to view |c3|, network connectivity will need to be available between your local machine and the Kubernetes pod running the |c3| service.  If you used an existing cluster you may already have external cluster access configured.  If you used the demo ``gke-create-cluster`` function, you can use the following ``kubectl`` command to open a forwarded port connection between your local host and |c3|.
+
+.. sourcecode:: bash
+
+		kubectl -n operator port-forward controlcenter-0 12345:9021
+
+Now open a web-browser to http://localhost:12345, and you should see |c3| with operational |ak| clusters, |sr|, and |kconnect-long|.
+
+.. figure:: images/c3.png
```

Comment:
1. Can we get the topic **inspection** view, instead of the topic overview?
2. For consideration, add a view that shows the topic schema (nudge folks to use Schema Registry)

Reply:
@ybyzek updated w/ new screenshots, let me know what you think of those